src/index.js

```diff
@@ -15,6 +15,18 @@ submitScore.addEventListener('click', (event) => {
   }
 });
 
+document.addEventListener('keypress', (event) => {
+  if (event.keyCode === 13 || event.which === 13) {
+    event.preventDefault();
+    if (document.getElementById('name').value === '' || document.getElementById('score').value === '') {
+      document.getElementsByClassName('error-message')[0].style.display = 'block';
+    } else {
+      document.getElementsByClassName('error-message')[0].style.display = 'none';
```

Comment:
Accessing DOM elements are a very heavy process and having many of them in our script cause the issue for performance, so please kindly consider to store the reference to a DOM element in a variable and reuse it instead of accessing many times using `document.quesrySelectior()` or any other selector. 🎯